### ISSUE TITLE
Controllable graph physics

### ISSUE THREAD

**Issue by d47853** [gchq/gaffer-tools#579]
Users should be able to define how they'd like their graph to look (changing the graph's physical properties). Here's an [example](http://js.cytoscape.org/demos/colajs-graph)

**Comment by n3101** [comment]
The UI fails to meet our Accessibility policy. It is unmaintainable and will be removed shortly.
